rules/no-native-promise-helpers.js

```diff
@@ -2,6 +2,8 @@
 
 const HELPERS = ['all', 'race', 'allSettled'];
 
+const TASK_TYPES = ['task', 'restartableTask', 'dropTask', 'keepLatestTask', 'enqueueTask'];
```

Comment:
maybe we should define this in a shared file somewhere instead of putting it in every rule file?